src/JustEat.StatsD/Transport/IStatsDTransport.cs

```diff
@@ -2,6 +2,6 @@ namespace JustEat.StatsD
 {
     public interface IStatsDTransport
     {
-        void Send(string metric);
+        void Send(in Data metric);
```

Comment:
Problem with this is that we're doing yet another breaking interface change and have to go to `4.0.0` when we've barely just released `3.0.0`.

Reply:
Yep, it hurts. The other option is to introduce v2 Transport interface. I'm not sure what is worse.